### ISSUE TITLE
No tabindex for 'Remember Me' at login, lack of keyboard toggle

### ISSUE THREAD

**Issue by seanosullivanuk** [BookStackApp/BookStack#1476]
The 'Remember Me' checkbox on the login page is an SVG item, which doesn't yet have a _tabindex_ attribute for keyboard navigation. Support for this is rudimentary, but having it within the markup would at least provide the functionality for users of Chrome, Edge Insider and Firefox.

Also, I use the space bar to toggle check boxes, but being an SVG this doesn't work. For wider compatibility and accessibility, may I request the SVG element changed to a checkbox input type?

**Comment by ssddanbrown** [comment]
Thanks for reporting @seanosullivanuk, This goes well with the intended accessibility improvements for the next release. 

The checkbox itself is actually a styled span inside a label, the SVG is just the check mark itself.
I've now updated the use of custom checkboxes to be tab-focusable and to have keyboard toggle functionality in addition to now having aligned accessibility markup.

In an ideal world we'd use normal checkboxes but they're unfortunately variant across operating systems while being hard to customize directly. 

These changes will be in the next patch release.
